<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panel_ntp_client</class>
 <widget class="QWidget" name="panel_ntp_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>服务器IP：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="ipLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>端口地址：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="portLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startPushButton">
           <property name="text">
            <string>启动(&amp;O)</string>
           </property>
           <property name="icon">
            <iconset resource="sim_iec61850.qrc">
             <normaloff>:/sim_iec61850/Resources/TIMER01.ICO</normaloff>:/sim_iec61850/Resources/TIMER01.ICO</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopPushButton">
           <property name="text">
            <string>停止(&amp;C)</string>
           </property>
           <property name="icon">
            <iconset resource="sim_iec61850.qrc">
             <normaloff>:/sim_iec61850/Resources/del.gif</normaloff>:/sim_iec61850/Resources/del.gif</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="timeSynPushButton">
           <property name="text">
            <string>对时(&amp;T)</string>
           </property>
           <property name="icon">
            <iconset resource="sim_iec61850.qrc">
             <normaloff>:/sim_iec61850/Resources/clock.gif</normaloff>:/sim_iec61850/Resources/clock.gif</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>244</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="sim_iec61850.qrc"/>
 </resources>
 <connections/>
</ui>
